flipperdevices/Flipper-Android-App
# Remotecontrols/save files api

**Background**

Sometimes app can crash when saving files on remote controls screen

**Changes**

- Replaces saving files with FlipperStorageApi

**Test plan**

- Open remote controls setup or remote control screen
- See files are saving as previous without crash
- Optionally delete temp folder
